src/components/Buckets.tsx

```diff
@@ -148,6 +151,18 @@ const Buckets = (): JSX.Element => {
               </Form.Select>
             </InputGroup>
           </Col>
+          <Col lg={3}>
```

Comment:
It would be better to have a similar search field design between apps and buckets